qemu/tests/block_stream.py

```diff
@@ -11,12 +11,22 @@
 class BlockStreamTest(blk_stream.BlockStream):
 
     def get_image_size(self, image_file):
-        qemu_img = utils_misc.get_qemu_img_binary(self.params)
-        cmd = "%s info %s" % (qemu_img, image_file)
-        info = utils.system_output(cmd)
-        size = re.findall("(\d+) bytes", info)
+        try:
+            qemu_img = utils_misc.get_qemu_img_binary(self.params)
+            cmd = "%s info %s" % (qemu_img, image_file)
+            info = utils.system_output(cmd)
+            size = int(re.findall("(\d+) bytes", info)[0])
+        except error.CmdError, err:
```

Comment:
In avocado,  `avocado/utils/process.py`,  there are `system_output` and `CmdError`, let's  try replace `error` and `utils.system_output`  with them since we change these code now.